py/selenium/webdriver/chrome/webdriver.py

```diff
@@ -76,6 +76,43 @@ def launch_app(self, id):
         """Launches Chrome app specified by id."""
         return self.execute("launchApp", {'id': id})
 
+    def get_network_conditions(self):
+        """
+        Gets Chrome network emulation settings.
+
+        Returns:
```

Comment:
This docstring is causing the docs build to fail. Run `tox -c py/tox.ini -e docs` and make sure they are passing.

Also update the docstrings to use `:Args:`, `:Usage:`, and `:Returns:` to be consistent. (See webdriver for example.)[https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/webdriver.py]